### PR TITLE
automatically create a requirements.txt 

### PR DESCRIPTION
**Pull request by ASHNA-AGGARWAL-KEYSIGHT**
resolves https://github.com/open-traffic-generator/openapiart/issues/302

### CODE REVIEW

**Review by Rangababu-R — DISMISSED** [review]

File: do.py [hunk 1]
```diff
@@ -166,6 +167,60 @@ def generate():
     )
 
 
+def generate_requirements():
+    lib_path = os.path.dirname(__file__)
```
[the remark this answers is not in the document]
Comment (ASHNA-AGGARWAL-KEYSIGHT) [reply do.py]: Added the changes

File: openapiart/generate_requirements.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            fh.write("\n")
+            for pkg in new_packages:
+                fh.write(pkg + "\n")
+            fh.flush()
+            packages = fh.read().splitlines()
+            fh.close()
+
+
+def generate_test_requirements(save_path):
+    not_required_pkgs = ['grpc', 'grpcio', 'grpcio-tools', 'protobuf']
```
Comment (ANISH-GOTTAPU) [inline openapiart/generate_requirements.py:60]: Add comment for this function

Comment (ANISH-GOTTAPU) [reply openapiart/generate_requirements.py:60]: CI/CD is failing, check once

File: generate_requirements.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        To generate the requirements.txt of library in its path
+    '''
+    if save_path is None:
+        file_name = "new_requirements.txt"
+        save_path = path
+
+    new_save_path = os.path.join(save_path, file_name)
+
+    if sys.version_info[0] < 3:
+        print("For versions less than 3 requirements.txt will not be generated")
```
Comment (ANISH-GOTTAPU) [inline generate_requirements.py:16]: NIT - Use `Python Versions` or `Python3`

**Review by ANISH-GOTTAPU — APPROVED** [review]

**Review by ANISH-GOTTAPU — APPROVED** [review]

**Pull request merged**